src/lib/twizzler-abi/src/runtime/alloc.rs

```diff
@@ -24,14 +30,33 @@ impl OomHandler for MinimalOomHandler {
         if layout.size() > ALLOC_OBJ_REG_SIZE {
             return Err(());
         }
-        let obj = InternalObject::<u8>::create_data_and_map().ok_or(())?;
+
+        let id = sys_object_create(
+            ObjectCreate::new(
+                BackingType::Normal,
+                LifetimeType::Volatile,
+                None,
+                ObjectCreateFlags::empty(),
+            ),
+            &[],
+            &[],
+        )
```

Comment:
I would really love a descriptive set of comments here.  There is clearly deep magic going on but it's not apparent to me just what that magic might be.